spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/jmx/JmxPropertiesConfigAdapter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.actuate.autoconfigure.metrics.export.jmx;
+
+import java.time.Duration;
+
+import io.micrometer.jmx.JmxConfig;
+
+import org.springframework.boot.actuate.autoconfigure.metrics.export.PropertiesConfigAdapter;
+
+/**
+ * Adapter to convert {@link JmxProperties} to a {@link JmxConfig}.
+ *
+ * @author Jon Schneider
+ */
+class JmxPropertiesConfigAdapter extends PropertiesConfigAdapter<JmxProperties>
+		implements JmxConfig {
+
+	JmxPropertiesConfigAdapter(JmxProperties properties) {
+		super(properties);
+	}
+
+	@Override
+	public String get(String k) {
+		return null;
```

Comment:
Is this normal?

Reply:
Yes. For non-Spring consumers of Micrometer, the `get` method provides a convenient way to bind the config interface to a property system with a certain prefix. Isn't useful for Spring because of how prescriptive we are about properties coming from classes with a certain annotation with bean-style getters, etc.